src/browser/extension/options/MiscellaneousGroup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default ({ options, saveOption }) => {
+  const browserName = typeof chrome === 'undefined' ? 'Firefox' : 'Chrome';
+
+  return (
+    <fieldset className="option-group">
+      <legend className="option-group__title">Miscellaneous</legend>
+
+      <div className="option option_value_max-age">
+        <label className="option__label" htmlFor="maxAge">Limit the action history to</label>
+        {' '}
+        <input className="option__element"
+               id="maxAge"
+               type="number"
+               min="1"
```

Comment:
I made the `maxAge` input a `number` input (so it has nice buttons to change the value with mouse) and limited it down to 1. Is the limit OK?